build.gradle

```diff
@@ -53,38 +57,34 @@ sourceSets {
 }
 
 dependencies {
-    implementation deps.kotlin.jdk
+    api deps.kotlin.jdk
+
+    api deps.logs.logback
+    api deps.logs.logbackJson
+    api deps.logs.coroutinesSlf4j
 
-    implementation deps.logs.logback
-    implementation deps.logs.logbackJson
-    implementation deps.logs.coroutinesSlf4j
+    api deps.ktor.serverNetty
+    api deps.ktor.serialization
+    api deps.ktor.auth
 
-    implementation deps.ktor.serverNetty
-    implementation deps.ktor.serialization
-    implementation deps.ktor.auth
+    api deps.koin.core
+    api deps.koin.coreExt
+    api deps.koin.ktor
 
-    implementation deps.koin.core
-    implementation deps.koin.coreExt
-    implementation deps.koin.ktor
+    api deps.valiktor
 
-    implementation deps.valiktor
-    // Database stuff
-    implementation deps.postgresql
-    implementation deps.hikari
-    implementation deps.exposed.core
-    implementation deps.exposed.dao
-    implementation deps.exposed.jdbc
-    implementation deps.flyaway
+    // Database
+    api deps.exposed.core
+    api deps.exposed.dao
+    api deps.exposed.jdbc
 
+    // Tests
     testImplementation deps.ktor.test
     testImplementation deps.koin.test
     testImplementation deps.junit.api
     testImplementation deps.junit.params
     testRuntimeOnly deps.junit.engine
     testImplementation deps.kluent
-    testImplementation deps.testContainers.core
```

Comment:
Removed test containers and postgresql dependencies as they should be imported by the consumers of the library.